crates/re_sdk/src/recording_stream.rs

```diff
@@ -916,7 +916,7 @@ impl RecordingStream {
             None
         } else {
             Some(DataRow::from_cells(
-                row_id,
+                row_id.incremented_by(1), // we need a unique RowId from what is used for the splatted data
```

Comment:
Nit: this would read less surprisingly if we moved the whole code block down below the `let splatted = `

Creating then in one order and logging them in the reverse order is a recipe for this exact kind of confusion.

Reply:
Good idea!